corehq/apps/domain/tests/test_domain_transfer.py

```diff
@@ -184,7 +184,8 @@ def test_permissions_for_transfer_domain_view(self):
         # No one logged in
         resp = self.client.get(reverse('transfer_domain_view',
                                        args=[self.domain.name]), follow=True)
-        self.assertEqual(resp.status_code, 404)
+        self.assertRedirects(resp, reverse('domain_login', kwargs={'domain': self.domain.name}) +
+            '?next=' + reverse('transfer_domain_view', args=[self.domain.name]))
```

Comment:
@sravfeyn Better? I feel like there might be a cleaner way to include the get string, but urlencode swaps the slashes for url encoding, so that fails.

As an aside, the value of `follow` doesn't seem to affect the outcome of the test.